content/en/agent/guide/dual-shipping.md

```diff
@@ -0,0 +1,305 @@
+```
+
+{{% /tab %}}
+
+{{% tab "APM" %}}
+
+### YAML configuration
+In `datadog.yaml`: 
+```yaml
+apm_config:
+  [...]
+  additional_endpoints:
+    "https://mydomain.datadoghq.com":
+    - apikey2
+    - apikey3
+    "https://mydomain.datadoghq.eu":
+    - apikey4
+
+  profiling_additional_endpoints:
+    "https://mydomain.datadoghq.com":
+    - apikey2
+    - apikey3
+    "https://mydomain.datadoghq.eu":
+    - apikey4
+```
+
+### Environment variable configuration
+
+```bash
+DD_APM_ADDITIONAL_ENDPOINTS='{\"https://mydomain.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://mydomain.datadoghq.eu\": [\"apikey4\"]}'
+
+DD_APM_PROFILING_ADDITIONAL_ENDPOINTS='{\"https://mydomain.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://mydomain.datadoghq.eu\": [\"apikey4\"]}'
+```
+
+{{% /tab %}}
+
+{{% tab "Live Processes" %}}
+
+### YAML configuration
+In `datadog.yaml`: 
+```yaml
+process_config:
+  [...]
+  additional_endpoints:
+    "https://mydomain.datadoghq.com":
+    - apikey2
+    - apikey3
+    "https://mydomain.datadoghq.eu":
+    - apikey4
+```
+
+### Environment variable configuration
+
+```bash
+DD_PROCESS_ADDITIONAL_ENDPOINTS='{\"https://mydomain.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://mydomain.datadoghq.eu\": [\"apikey4\"]}'
+```
+
+{{% /tab %}}
+
+{{% tab "Orchestrator" %}}
+
+### YAML configuration
+In `datadog.yaml`: 
+```yaml
+orchestrator_explorer:
+  [...]
+  orchestrator_additional_endpoints:
+    "https://mydomain.datadoghq.com":
+    - apikey2
+    - apikey3
+    "https://mydomain.datadoghq.eu":
+    - apikey4
+```
+
+### Environment variable configuration
+
+```bash
+DD_ORCHESTRATOR_EXPLORER_ORCHESTRATOR_ADDITIONAL_ENDPOINTS='{\"https://mydomain.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://mydomain.datadoghq.eu\": [\"apikey4\"]}'
+```
+
+{{% /tab %}}
+{{% /tabs %}} 
+
+## Logs, Database Monitoring, Network Devices, CSPM, Runtime Security
+
+For data from these products, when setting up additional endpoints, you must explicitly set `use_http` to tell the Agent which transport to use. The same transport configuration is shared among all additional endpoints.
+
+The `is_reliable` setting tells the Agent to treat this endpoint with the same priority as the primary endpoint. The primary endpoint is always reliable. The `is_reliable` setting ensures that logs are not missed if a destination becomes unavailable. 
+
+For example, if you're sending logs to two reliable endpoints and one becomes unavailable, logs continue to flow to the second reliable endpoint. If both endpoints become unavailable, the logs stop flowing until at least one endpoint recovers.
+
+The `is_reliable` setting defaults to `false`. Unreliable endpoints only send logs if they successfully send to at least one reliable endpoint. You may define multiple additional endpoints with mixed use of `is_reliable`. 
```

Comment:
Could you clarify this, or rephrase? It's not super clear.

Reply:
I tried to clarify as best I could. It's a complicated behavior so it's difficult to summarize in simple terms. Let me know if this is better.